README.md

```diff
@@ -678,6 +678,6 @@ There is no configuration required for the UserDataPlugin as it just handles req
 
 ## License
 
-Copyright (c) 2022 [SpringRoll](http://github.com/SpringRoll)
+Copyright (c) 2024 [SpringRoll](http://github.com/SpringRoll)
```

Comment:
👀 is this something we need to update in all the SpringRoll plugins?

Reply:
To be honest I have no idea. Its just something I update whenever I notice it